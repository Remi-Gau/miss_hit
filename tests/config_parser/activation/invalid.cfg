enable: 1.0
enable: potato
enable: "true"
enable: if
enable: enable
enable: 1 enable: 0
enable: 2
enable: -1
enable: # potato
   true
