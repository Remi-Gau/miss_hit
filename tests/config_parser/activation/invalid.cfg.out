In invalid.cfg, line 1
| enable: 1.0
|         ^^^ error: expected an integer number
In invalid.cfg, line 2
| enable: potato
|         ^^^^^^ error: expected true or false
In invalid.cfg, line 3
| enable: "true"
|         ^^^^^^ error: expected true or false
In invalid.cfg, line 4
| enable: if
|         ^^ error: expected true or false
In invalid.cfg, line 5
| enable: enable
|         ^^^^^^ error: expected true or false
In invalid.cfg, line 6
| enable: 1 enable: 0
|           ^^^^^^ error: expected NEWLINE, found IDENTIFIER instead
In invalid.cfg, line 7
| enable: 2
|         ^ error: expected true or false
In invalid.cfg, line 8
| enable: -1
|         ^ error: expected true or false
In invalid.cfg, line 9
| enable: # potato
|                 ^ error: expected true or false
invalid.cfg: error: config file contains errors
MISS_HIT Debug Summary: 1 file(s) analysed, 10 error(s)
